File: app/src/main/java/com/mobile/rxjava2andretrofit2/mine/view/IMineView.java
package com.mobile.rxjava2andretrofit2.mine.view;

import com.mobile.rxjava2andretrofit2.base.IBaseView;
import com.mobile.rxjava2andretrofit2.mine.bean.MineResponse;

import java.util.List;

public interface IMineView extends IBaseView {

    void mineDataSuccess(List<MineResponse.AnsListBean> success);

    void mineDataError(String error);

}

File: app/src/main/java/com/mobile/rxjava2andretrofit2/kotlin/mine/view/IMineView.kt
package com.mobile.rxjava2andretrofit2.kotlin.mine.view

import com.mobile.common_library.base.IBaseView
import com.mobile.rxjava2andretrofit2.kotlin.mine.bean.Ans

interface IMineView : IBaseView {

    fun mineDataSuccess(success: List<Ans>)

    fun mineDataError(error: String)

}